services/postgres/requirements.yaml

```diff
@@ -0,0 +1,4 @@
+dependencies:
+- name: postgres
+  version: ">=0.0.1"
```

Comment:
I've seen this done two ways: pinning the version and requiring a change to bump it (what I've been doing), and using this syntax so that new Helm charts are picked up automatically. Should we standardize on one approach or the other?

Reply:
Until I have it working, I want to let it float.  Once it's working we should definitely pin it for stable, anyway.